praw/models/reddit/draft.py

```diff
@@ -0,0 +1,331 @@
+            "flair_text": flair_text,
+            "is_public_link": is_public_link,
+            "kind": "markdown" if selftext is not None else "link",
+            "nsfw": nsfw,
+            "original_content": original_content,
+            "send_replies": send_replies,
+            "spoiler": spoiler,
+            "title": title,
+        }
+        if subreddit:
+            data.update(
+                {
+                    "subreddit": subreddit.fullname,
+                    "target": "profile"
+                    if subreddit.display_name.startswith("u_")
+                    else "subreddit",
+                }
+            )
+        data.update(draft_kwargs)
+        data = {key: value for key, value in data.items() if value is not None}
+        return data
+
+    def __init__(
+        self,
+        reddit: "praw.Reddit",
+        id: Optional[str] = None,  # pylint: disable=redefined-builtin
+        _data: Dict[str, Any] = None,
+    ):
+        """Construct an instance of the Draft object."""
+        if (id, _data).count(None) != 1:
+            raise TypeError("Exactly one of `id` or `_data` must be provided.")
+        fetched = False
+        if id:
+            self.id = id
+        else:
+            if len(_data) > 1:
+                if _data["kind"] == "markdown":
+                    _data["selftext"] = _data.pop("body")
+                elif _data["kind"] == "link":
+                    _data["url"] = _data.pop("body")
+                fetched = True
+        super().__init__(reddit, _data=_data, _fetched=fetched)
+
+    def __eq__(self, other: "Draft"):
+        """Return whether the other instance equals the current."""
+        if isinstance(other, str):
+            return other.lower() == str(self).lower()
+        body = True
+        if self._fetched and other._fetched:
+            if self.kind == other.kind == "markdown":
+                body = self.selftext == other.selftext
+            elif self.kind == other.kind == "link":
+                body = self.url == other.url
+        return all(
+            [
+                isinstance(other, self.__class__),
+                self.id.lower() == other.id.lower(),
+                body,
+            ]
+        )
+
+    def __hash__(self):  # not sure why this doesn't inherit correctly
+        """Return the hash of the current instance."""
+        return super().__hash__()
+
+    def __repr__(self) -> str:
+        """Return a string representation of the Draft instance."""
+        if self._fetched:
+            subreddit = (
+                f" subreddit={self.subreddit.display_name!r}" if self.subreddit else ""
+            )
+            title = f" title={self.title!r}" if self.title else ""
+            return f"{self.__class__.__name__}(id={self.id!r}{subreddit}{title})"
+        else:
+            return f"{self.__class__.__name__}(id={self.id!r})"
+
+    def _fetch(self):
+        for draft in self._reddit.drafts():
```

Comment:
Is there no endpoint to directly access a single draft?

Reply:
There is not. Reddit only fetches all of them on the website and app.